### PR TITLE
LPS-162619

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-162619

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp [hunk 1]
```diff
@@ -245,6 +245,12 @@ renderResponse.setTitle(LanguageUtil.format(request, "edit-x", objectDefinition.
 						<aui:input disabled="<%= objectDefinition.isSystem() || !objectDefinitionsDetailsDisplayContext.hasUpdateObjectDefinitionPermission() %>" label="" labelOff='<%= LanguageUtil.get(request, "enable-comments") %>' labelOn='<%= LanguageUtil.get(request, "enable-comments") %>' name="enableComments" type="toggle-switch" value="<%= objectDefinition.isEnableComments() %>" />
 					</aui:field-wrapper>
 				</c:if>
+
+				<c:if test='<%= GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-158473")) && objectDefinition.isDefaultStorageType() %>'>
+					<aui:field-wrapper cssClass="form-group lfr-input-text-container">
+						<aui:input disabled="<%= objectDefinition.isSystem() || !objectDefinitionsDetailsDisplayContext.hasUpdateObjectDefinitionPermission() %>" label="" labelOff='<%= LanguageUtil.get(request, "enable-entry-history") %>' labelOn='<%= LanguageUtil.get(request, "enable-entry-history") %>' name="enableEntryHistory" type="toggle-switch" />
```
Comment (rodrigopaulino) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp:251]: @alinedoleron, we're not going to have a specific permission for that. Could you remove the hasUpdateObjectDefinitionPermission verification and include one that disables the switch if the definition is published (active)?

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hi, @alinedoleron.
Nice job. We have some changes regarding the business rules, so could you please verify them below?
Also, can you change the order of the commits related to the language keys? The inclusion of the key should come before the buildLang. Someone rebasing this wouldn't be able to regen the keys in this order in case of a conflict because the key is only introduced after the first commit.
Thx!

**Review by rodrigopaulino — APPROVED** [review]
Nice, @alinedoleron.
Thanks a lot.
LGTM 🎉

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by rodrigopaulino** [comment]
@alinedoleron, could you please manually forward this to Brian and mention that the failure is not related to the PR?
Thx.

**Comment by alinedoleron** [comment]
Pr forwarded: https://github.com/brianchandotcom/liferay-portal/pull/123568

**Pull request closed without merging**